src/dataset-builder/DatasetBuilder.test.ts

```diff
@@ -258,7 +258,7 @@ describe('DatasetBuilder', () => {
     const user = userEvent.setup();
     await initializeValidDatasetRequest(user);
     // Assert
-    expect(await screen.findByText('100 Participants in this dataset')).toBeTruthy();
+    expect(await screen.findByText('100 Participants in this dataset', {}, { timeout: 1000 })).toBeTruthy();
```

Comment:
this doesn't work! this test fails. Trying to debug it because I can see that 100 Participants in this dataset does exist on the screen. It is divided into 100, Participants in this dataset but it has been this way and the test was passing. This test passes for just 'Participants in this dataset' but not just '100'. I think it is because it is loaded data now so it is not immediately on the page maybe. But the timeout does not help, so maybe not. And, another problem with this is it will not always be 100 once the endpoint is implemented. So, this should be changed to match any number, but I'm not sure how to do that.